org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsCurrent.java

```diff
@@ -72,12 +72,8 @@ public boolean initializeDevice() {
             return false;
         }
 
-        divisor = serviceClusterMeasurement.getAcCurrentDivisor(Long.MAX_VALUE);
-        multiplier = serviceClusterMeasurement.getAcCurrentMultiplier(Long.MAX_VALUE);
-        if (divisor == null || multiplier == null) {
-            divisor = 1;
-            multiplier = 1;
-        }
+        determineDivisorAndMultiplier(serverClusterMeasurement);
```

Comment:
Why do we need to call this here? It's right at the end of the initialisation method, so anything done there is not used.

Reply:
Will remove it.